apps/angular/src/app/features/anime-dashboard/anime-dashboard.component.ts

```diff
@@ -110,9 +112,17 @@ export class AnimeDashboardComponent {
 		this.navigate(filterParams);
 	}
 
+	/**
+	 * Triggers when an anime is selected.
+	 * @param event - Selected anime index.
+	 */
+	protected onAnimeSelect(event: number): void {
```

Comment:
```suggestion
	protected onAnimeSelect(animeId: number): void {
```
We prefer to use a readable name for the argument if possible.
We prefer to use the word event only for an argument with the Event type